test/suite/test_cursor22.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# test_cursor22.py
+#   Test cursor get_raw_key_value
+
+import wttest
+from wtscenario import make_scenarios
+from wiredtiger import stat
+
+class test_cursor22(wttest.WiredTigerTestCase):
+    uri = "table:test_cursor22"
```

Comment:
Could you extend this to work with different table types (row, column, fixed length column).
I'm not sure if it's also possible to use the API with a complex schema, but if it is, it would be good to add test coverage to that here.
Something like the scenarios described in test_cursor09 is what I'm thinking.

Reply:
I've created a new test (test_cursor23) based on some of the scenarios in test_cursor09. Is that the kind of thing you had in mind? The 'complex' example shows/tests that this new API is unsupported on such examples.

Reply:
So I've added two tests: 
- test_cursor22 which is deliberately as simple as possible, and 
- test_cursor23 which addresses more complex scenarios.